contracts-periphery/foundry.toml

```diff
@@ -4,4 +4,7 @@ out = 'out'
 libs = ['lib']
 remappings = ['ds-test/=lib/ds-test/src/', '@openzeppelin/=lib/openzeppelin-contracts']
```

Comment:
Do we need this ds-test mapping?

Also thoughts on removing the openzeppelin one too and just using the actual path instead of the `@` remapping? I don't think the custom `@` syntax buys us anything, and is kind of just an artifact of npm/hardhat resolution

Reply:
Looks clean without them. I think it's there just so if you want to deploy using hardhat, you don't have to change the imports.

Reply:
Might have to keep the ds-test one tho because dependencies like forge-std imports uses it even though the build didn't fail..

Reply:
We don't need to keep the ds-test one either. I tried deleting it and everything still worked.

However, I personally like to add 
```toml
[default]
remappings = ["test/=test/"] # Allows imports using `test/` to refer to root test directory.
```

this way we can import all paths with absolute paths, e.g. `import src/File.sol` and `import test/File.t.sol` instead of using relative paths

Reply:
Sounds good, fixed it here https://github.com/ScopeLift/umbra-protocol/pull/334/commits/191847973022f7850f017907571562d023f4918b

Reply:
I also just pushed 2f5c656 to update our dep versions and 3b85fb1 to leverage your commit to use absolute paths instead of relative ones